build-system/amp.extern.js

```diff
@@ -107,6 +107,33 @@ ExtensionPayload.prototype.i;
  */
 var JsonValue;
 
+/**
+ * @constructor
+ * @dict
+ */
+function VideoAnalyticsDetailsDef() {};
```

Comment:
@aghassemi please review this part

Reply:
matches existing `VideoAnalyticsDetailsDef` LGTM